asmcli/cloudbuild-main.yaml

```diff
@@ -133,6 +133,24 @@ steps:
     #############
     # SLOW TESTS
     #############
+  - name: 'gcr.io/$PROJECT_ID/${_IMAGE_NAME}'
+    dir: 'asmcli'
+    id: 'run-create-mesh'
+    entrypoint: '/bin/bash'
+    args:
+      - '-c'
+      - >
+        ./tests/run_create_mesh
+        --PROJECT_ID "${PROJECT_ID}"
+        --BUILD_ID "${BUILD_ID}"
+    env:
+      - 'SERVICE_ACCOUNT=${_SERVICE_ACCOUNT}@${PROJECT_ID}.iam.gserviceaccount.com'
+      - 'KEY_FILE=${_KEY_FILE}'
+      - '_CI_ASM_IMAGE_LOCATION=${_IMAGE_LOCATION}'
+      - '_CI_ASM_PKG_LOCATION=${_PKG_LOCATION}'
+      - '_CI_ASM_KPT_BRANCH=${_ASM_PKG_BRANCH}'
+    timeout: 600s
+    
```

Comment:
It's probably a good idea to run it on this PR since it's refactoring, but let's not add another slow test to the `main` file. Can you put this in the `staging` version before merging?

Reply:
yes that was my plan. Just want to wait till it's approved. 😉